src/Network/Network/ChangeLog.md

```diff
@@ -21,9 +21,16 @@
 
 ## Version 1.1.0
 * Update incorrect online help URLs
+* Added Cmdlets for Identity on Application Gateway.
```

Comment:
@akshaysngupta the content you added here should be under the `Upcoming Release` header

Reply:
corrected.

Reply:
@akshaysngupta same comment -- no new commit, so this is still an issue